### DISCUSSION TITLE
How to modify the inheritance (convert one object to another?)

### DISCUSSION THREAD

**Discussion by humbertolopezleon** [jazzband/django-polymorphic#730] [Q&A]
I am using **djangot-polymorphic** and so far it worked fine, but I don't know how to convert an object from one type to another, that is, _modify the inheritance_. I thought that updating the value of the resourcetype would solve the problem but it is not true. Do you have any idea how to get this?

**Comment by bckohan** [comment]
@humbertolopezleon If you mean, modify the class hierarchy - then you will need to:

1. Create the new classes
2. Create a migration to copy the old rows into the new db rows
3. Delete the old classes

If you mean promote an existing row down the hierarchy, we added a [`create_from_super` function in 4.4.0](https://django-polymorphic.readthedocs.io/en/latest/api/polymorphic.managers.html#polymorphic.managers.PolymorphicManager.create_from_super).
